packages/frontend/src/config/environmentDefaults/mainnet.ts

```diff
@@ -37,6 +37,8 @@ export default {
         '8DKTSceSbxVgh4ANXwqmRqGyPWCuZAR1fCqGPXUjD5nZ',
         // https://github.com/near/core-contracts/blob/f93c146d87a779a2063a30d2c1567701306fcae4/multisig/res/multisig.wasm
         '55E7imniT2uuYrECn17qJAk9fLcwQW4ftNSwmCJL5Di',
+        // unknown
+        'HRP7Qf2HDaXTD8EWs7G8siNNGxWWBCe1JaDxcM2cJmQR',
```

Comment:
where you got the hash?

Reply:
`NEAR_ENV=mainnet near state multisig.pierre-dev.near`

Run this command in terminal and get the code hash... Every time smart contract is edited, even minor edit will cause the code hash to become different.

The `core-contracts/multisig` repo in git has undergone so many versions, so there is a lot of multisign hashes there.
https://github.com/near/core-contracts/tree/master/multisig

As long as it is a multisig contract hashes, it should be recognised by MNW.